app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+import random
+import matplotlib.pyplot as plt
+
+
+def flip_coin():
+    case = ["head", "tail"]
+    result_list = []
+    for _ in range(10000):
+        head_count = 0
+        for _ in range(10):
+            result = random.choice(case)
+            if result == "head":
+                head_count += 1
+
+        result_list.append(head_count)
+
+    results = {key: 0 for key in range(0, 11)}
+
+    for result in result_list:
+        results[result] += 1
+
+    return results
```

Comment:
It should return dict, where keys are numbers of possible heads dropped (0 to 10), and values are percentage of how many that number of heads dropped out of all cases.